src/coreclr/tools/aot/ILCompiler.ReadyToRun/Interop/IL/Marshaller.ReadyToRun.cs

```diff
@@ -25,12 +25,15 @@ protected static Marshaller CreateMarshaller(MarshallerKind kind)
                     return new BooleanMarshaller();
                 case MarshallerKind.AnsiString:
                     return new AnsiStringMarshaller();
+#if !READYTORUN
```

Comment:
just realized that this is specific to R2R so just need to remove these instead of `#ifdef`-ing